test/Microsoft.ML.CodeAnalyzer.Tests/Code/BestFriendTest.cs

```diff
@@ -66,8 +66,9 @@ public void BestFriend()
                 peRef = MetadataReference.CreateFromImage(peImage);
             }
 
+            refs.ToList().Add(peRef);
             var comp = projB.GetCompilationAsync().Result
-                .WithReferences(refs.Append(peRef).ToArray());
+                .WithReferences(refs.ToArray());
```

Comment:
Part of the PR?

Reply:
yes append function is not available on .netframwork(This project was earlier targeting the .netcoreapp2.1)

Reply:
The `Append` method is one of those rare APIs that is in `netstandard2.0` but we didn't ship in `net461`. We had a similar bug in the product - see https://github.com/dotnet/machinelearning/issues/1072